### ISSUE TITLE
JSON formatting for programatic use

### ISSUE THREAD

**Issue by XenotropicDev** [rsek/datasworn#1]
The formatting of the JSON as well as the lack of a "name" property makes it hard to use the raw JSON as the source for importing the data using conventional JSON deseralizers.

The following snippet requires a "Starship" object and a "Engine Upgrade" object. Then for a program to use them generically a custom class/interface/object needs to be added, and then each item needs to be converted to that custom class.
```  
"Assets": {
    "Starship": {
      "Asset Type": "Command Vehicle"
      },
     "Engine Upgrade": {
      "Asset Type": "Module"
      }
     }
```

Where as the following format allows each item to use the built-in JSON object, and be accessed as a generic asset through the JSON assets array:
```  
"Assets": [
    {
    "Name": "Starship",
    "Asset Type": "Command Vehicle"
    },
     "Name":"Engine Upgrade",
     "Asset Type": "Module"
    }
    ]
```

It might be worthwhile to have a more natural language version (the current state), and a more programmatic version of each JSON file.

**Comment by rsek** [comment]
i've just added an alternate version of the asset data to dataforged. if that looks all right to you i can look at creating alternate versions for all of them

**Comment by rsek** [comment]
resolved in commit c1ff1d4f708c1f09003aa88d62e4d9bb57613fe4
